src/Sql/Sql/Server/Cmdlet/NewAzureSqlServer.cs

```diff
@@ -226,7 +226,7 @@ protected override IEnumerable<Model.AzureSqlServerModel> ApplyUserInputToModel(
                 SqlAdministratorLogin = (this.SqlAdministratorCredentials != null) ? this.SqlAdministratorCredentials.UserName : null,
                 Tags = TagsConversionHelper.CreateTagDictionary(Tags, validate: true),
                 Identity = ResourceIdentityHelper.GetIdentityObjectFromType(this.AssignIdentity.IsPresent, this.IdentityType ?? null, UserAssignedIdentityId, null),
-                MinimalTlsVersion = this.MinimalTlsVersion,
+                MinimalTlsVersion = (this.MinimalTlsVersion != null) ? this.MinimalTlsVersion : "1.2",
```

Comment:
Could the default value be empty?

Reply:
[ValidateSet("None", "1.0", "1.1", "1.2")] : These are the validated values.
It would not be empty.